src/sentry/api/event_search.py

```diff
@@ -144,7 +145,7 @@ def translate(pat):
 # Boolean comparison filter
 boolean_filter       = negation? search_key sep boolean_value
 # Aggregate numeric filter
-aggregate_filter          = negation? aggregate_key sep operator? (numeric_value / duration_format / percentage_format)
+aggregate_filter          = negation? aggregate_key sep operator? (duration_format / numeric_value / percentage_format)
```

Comment:
This should be fine since the aggregate filter function checks
`search_value.expr_name == "duration_format" and self.is_duration_key` so numeric keys shouldn't get parsed as a duration. They will get parse as numeric values by the [catch all](https://github.com/getsentry/sentry/pull/24261/files#diff-0b239bf9fccf786d1ac7bd5498e7133b828157098100ba72ceae29b2254ca915R487)